ipatests/util.py

```diff
@@ -709,8 +711,12 @@ def change_principal(user, password, client=None, path=None,
 
     try:
         with private_ccache(ccache_name):
-            kinit_password(user, password, ccache_name,
-                           canonicalize=canonicalize, enterprise=enterprise)
+            if keytab:
```

Comment:
Is here a typo or word missing? I do not understand. If one specifies keytab I would assume the tester knows what he's doing.

I can document the context manager though.